src/ImageSharp/Formats/Jpeg/Components/Block8x8F.cs

```diff
@@ -80,9 +80,9 @@ internal partial struct Block8x8F
             set => this[(y * 8) + x] = value;
         }
 
-        public static Block8x8F operator *(Block8x8F block, float value)
+        public static Components.Block8x8F operator *(Components.Block8x8F block, float value)
```

Comment:
I'm getting a "Qualifier is redundant" tip here.
I wonder which analyzer added the namespace qualifier?

Reply:
It was Resharper, probably due to the struct being a partial.